### DISCUSSION TITLE
HELP- Build error

### DISCUSSION THREAD

**Discussion by CvEngin** [eclipse-kura/kura#5326] [General]
[DEBUG] No solution found because the problem is unsatisfiable.: [Unable to satisfy dependency from org.eclipse.kura.util 1.6.0 to java.package; org.osgi.service.component.runtime 1.4.0.; Unable to satisfy dependency from org.eclipse.kura.util 1.6.0 to java.package; org.osgi.service.component.runtime.dto 1.4.0.; No solution found because the problem is unsatisfiable.]
[INFO] {osgi.os=win32, osgi.ws=win32, org.eclipse.update.install.features=true, osgi.arch=x86}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.eclipse.kura.asset.cloudlet.provider 1.5.0
[ERROR]   Missing requirement: org.eclipse.kura.util 1.6.0 requires 'java.package; org.osgi.service.component.runtime 1.4.0' but it could not be found
[ERROR]   Cannot satisfy dependency: org.eclipse.kura.asset.cloudlet.provider 1.5.0 depends on: java.package; org.eclipse.kura.util.collection [1.0.0,2.0.0)

**Comment by MMaiero** [comment]
Can you please give more info on where and when you are getting this error?

Thanks

**Reply by CvEngin** [reply]
Sorry, I encountered an issue during the process of building the target platform and was unable to set it up properly. I have resolved the problem and apologize for any inconvenience caused
